EventListenerManager.js

```diff
@@ -45,22 +45,25 @@ export default class EventListenerManager {
       return results.then(this.normalizeResults);
     else
       return this.normalizeResults(results);
-  }
+  };
```

Comment:
This change looks needless. Could you revert only this part?

Reply:
Sorry about that. I try not to change code I don't need to. I think I ran a code formatter to tidy up and forgot to drop this change. It's fixed now.

Reply:
Thanks!